src/commands/runtime/activation/list.js

```diff
@@ -62,40 +62,52 @@ class ActivationList extends RuntimeBaseCommand {
           },
           Kind: {
             get: (row) => {
-              const { annotations } = row
-              if (!annotations || !annotations.length) return
-              return annotations.find((elem) => {
-                return (elem.key === 'kind')
-              }).value
+              if (row.duration !== undefined) {
+                const { annotations } = row
+                if (!annotations || !annotations.length) return
+                return annotations.find((elem) => {
+                  return (elem.key === 'kind')
+                }).value
+              } else {
+                // this is a trigger
+                return 'trigger'
```

Comment:
Should this be `unknown` instead of `trigger` to be consistent with wsk cli ? I'm ok either way. Just an observation.

Reply:
If duration is not present, the activation is from a trigger. If you are seeing `unknown` from the `wsk` CLI then there's a bug in the latter.